app/lib/core/models/atr_identifier.dart

```diff
@@ -12,6 +12,16 @@ class AtrIdentifier {
     this.isComplete = false,
   });
 
+  factory AtrIdentifier.defaultAtrId({
+    String userId,
+    bool isComplete,
+  }) =>
+      AtrIdentifier(
+        //TODO: assign current userId to userId
```

Comment:
Actually, I think it would be the responsibility of the class creating the `AtrIdentifier` to give it the userID of the current user. I'm addressing this in #186 .